config/game_options.txt

```diff
@@ -699,3 +557,6 @@ MONKEYCAP 64
 
 ## Uncomment to use TG-style combat
 #DISABLE_STAMBUFFER
+
+#Replaces standard extended/secret dichotomy with extended and 5 levels of chaos using dynamic.
```

Comment:
comment needs to change

Reply:
fuixed in https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9557/commits/3a48b7238090181a90e93a6affbe25efd9423f32